src/style.css

```diff
@@ -2,2 +1,85 @@
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
+li,
+a {
+  text-decoration: none;
+}
```

Comment:
Don't use tag names for styling (except `html` and `body`). Style all elements using `.class` and if needed with `:pseudo-class`, `pseudo-element` and `[attribute]`